at_client/lib/src/client/at_client_spec.dart

```diff
@@ -17,9 +17,9 @@ abstract class AtClient {
   SyncManager? getSyncManager();
 
   /// Returns a singleton instance of [RemoteSecondary] to communicate with user's secondary server.
-  RemoteSecondary? getRemoteSecondary();
+  RemoteSecondary getRemoteSecondary();
 
-  LocalSecondary? getLocalSecondary();
+  LocalSecondary getLocalSecondary();
```

Comment:
So we're making it so that LocalSecondary is always present?

Reply:
As per the discussion, making LocalSecondary a non nullable instance.